cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -780,30 +787,6 @@ def test_self_reported_ethnicity_ontology_term_id__duplicates_in_multi_term(self
             ],
         )
 
-    def test_self_reported_ethnicity_ontology_term_id__multi_term_list(self):
```

Comment:
was this removed intentionally?

Reply:
didn't mean to push, I will re-add when I merge in the pytest PR